assets/templates/partials/census/collapsible.tmpl

```diff
@@ -1,20 +1,20 @@
-<div id="collapsible" class="collapsible js-collapsible collapsible--initialised u-pb-no" data-btn-close="{{ localise "HideThis" .Language 1 }}">
-    <div class="collapsible__heading js-collapsible-heading">
-        <div class="collapsible__controls">
-            <h2 class="collapsible__title u-mt-no">{{ .Title }}</h2>
-            <span class="collapsible__icon u-pt-xxs">
-                <svg class="svg-icon" viewBox="0 0 8 13" xmlns="http://www.w3.org/2000/svg" focusable="false">
+<div id="collapsible" class="ons-collapsible ons-js-collapsible ons-collapsible--initialised ons-u-pb-no" data-btn-close="{{ localise "HideThis" .Language 1 }}">
+    <div class="ons-collapsible__heading ons-js-collapsible-heading">
+        <div class="ons-collapsible__controls">
+            <h2 class="ons-collapsible__title ons-u-mt-no">{{ .Title }}</h2>
+            <span class="ons-collapsible__icon ons-u-pt-xxs">
+                <svg class="ons-svg-icon" viewBox="0 0 8 13" xmlns="http://www.w3.org/2000/svg" focusable="false">
                     <path d="M5.74,14.28l-.57-.56a.5.5,0,0,1,0-.71h0l5-5-5-5a.5.5,0,0,1,0-.71h0l.57-.56a.5.5,0,0,1,.71,0h0l5.93,5.93a.5.5,0,0,1,0,.7L6.45,14.28a.5.5,0,0,1-.71,0Z" transform="translate(-5.02 -1.59)"></path>
                 </svg>
             </span>
         </div>
     </div>
-    <div id="collapsible-content" class="collapsible__content js-collapsible-content">
+    <div id="collapsible-content" class="ons-collapsible__content ons-js-collapsible-content">
         {{ range $i, $v := .Content }}
-            <p class="u-fs-r u-mb-s u-p-no">{{ $v }}</p>
+            <p class="ons-u-fs-r ons-u-mb-s ons-u-p-no">{{ $v }}</p>
         {{ end }}
-        <button type="button" class="btn btn--small u-fs-r js-collapsible-button btn--secondary u-d-no u-p-no" aria-controls="collapsible">
-            <span class="btn__inner js-collapsible-button-inner u-fs-r">{{ localise "HideThis" .Language 1 }}</span>
+        <button type="button" class="ons-btn ons-btn--small ons-u-fs-r ons-js-collapsible-button ons-btn--secondary ons-u-d-no ons-u-p-no" aria-controls="collapsible">
+            <span class="ons-btn__inner ons-js-collapsible-button-inner ons-u-fs-r">{{ localise "HideThis" .Language 1 }}</span>
```

Comment:
Similar to above
```suggestion
            <span class="ons-btn__inner ons-js-collapsible-button-inner">{{ localise "HideThis" .Language 1 }}</span>
```

These changes will lead to this:

**Before**:
![image](https://user-images.githubusercontent.com/23668262/135403839-16f1b97a-05fb-4349-91da-c02e2d0d7009.png)


**After**:
![image](https://user-images.githubusercontent.com/23668262/135403813-8c03e2ae-5e80-42e5-895e-19a3ea86737b.png)